src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/Microsoft.Extensions.AI.Evaluation.Reporting.csproj

```diff
@@ -23,7 +23,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Include="..\TypeScript\html-report\dist\index.html" Visible="false">
+    <EmbeddedResource Condition="'$(EnableEvaluationReportBuild)'=='true'"
+        Include="..\TypeScript\html-report\dist\index.html" Visible="false">
```

Comment:
Is there some way to set this globally when building inside VS without having to leave a repo file checked out? I vaguely remember that MSBuild supports env vars as properties - would that work?

Reply:
Setting `EnvEnableEvaluationReportBuild` will now set this property and enable the report build.